packages/embeds/lib/Embed.ts

```diff
@@ -0,0 +1,198 @@
+import type { APIEmbed } from "@guildedjs/guilded-api-typings";
+
+import { resolveColor } from "./util";
+
+export class Embed {
```

Comment:
seems inconsistent that `public` is being explicitly declared everywhere in this class, since we don't do that anywhere else

Reply:
ah, yeah that is a remanent of the old selfbot code.